client/src/components/job-application-component/box-row-component/index.jsx

```diff
@@ -0,0 +1,145 @@
+import * as React from "react";
+import { useState } from "react";
+import { Grid, Typography, Select, MenuItem, Box, FormControl, InputLabel } from "@mui/material";
+import DownloadIcon from "@mui/icons-material/Download";
+import { IMaskInput } from "react-imask";
+
+import PropTypes from "prop-types";
+import { BoxDivider } from "./index.styles";
+
+const TextMaskCustom = React.forwardRef(function TextMaskCustom(
+  { onChange, name, ...other },
+  ref,
+) {
+  return (
+    <IMaskInput
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      {...other}
+      mask="(#00) 000-0000x0000"
+      definitions={{
+        "#": /[1-9]/,
+      }}
+      inputRef={ref}
+      onAccept={(value) => onChange({ target: { name, value } })}
+      overwrite
+    />
+  );
+});
+
+TextMaskCustom.propTypes = {
+  name: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+function BoxRowComponent({
+  leftSide,
+  rightSide,
+  downloadable,
+  editable,
+  setRightSide,
+  required,
+  options,
+}) {
+  const [selectedValue, setSelectedValue] = useState(rightSide[0] || "Select");
+
+  return (
+    <>
+      <Grid
+        container
+        justifyContent="space-between"
+        alignItems="center"
+        sx={{ mb: 1, paddingX: "20px" }}
```

Comment:
Super nit picky, but let's add slightly more vertical padding to make it match the Figma a little bit better 
Figma:
![image](https://github.com/user-attachments/assets/f68c78d3-234a-4148-9385-9627c66b48fd)
Yours:
![image](https://github.com/user-attachments/assets/2e1c515b-7d84-4930-b8a9-8e621b421b30)